Unit-01/03-templating/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+# PART 1
+@app.route('/person/<name>/<int:age>')
+def age(name, age):
+	return render_template('name.html', name = name, age = age)
+
+
+# PART 2
+@app.route('/calculate')
+def calculate():
+	return render_template('calculator.html')
+
+@app.route('/math')
+def math():
+	num1 = int(request.args.get("num1"))
+	num2 = int(request.args.get("num2"))
+	calculation = request.args.get("calculation")
```

Comment:
this section should be structured like this next time:

```python
@app.route("/math/<operation>/<int:num1>/<int:num2>")
```

Since we want to do the operation using the route instead of a query string